src/olympia/addons/tests/test_views.py

```diff
@@ -1408,12 +1407,12 @@ def get_category(type_, name):
 
         # Search for add-ons in the first category. There should be two.
         data = self.perform_search(self.url, {'app': 'firefox',
-                                              'type': 'search,statictheme',
-                                              'category': 'music'})
+                                              'type': 'extension,statictheme',
+                                              'category': 'other'})
```

Comment:
Should we use a variable to store this category slug instead of relying on it being hardcoded correctly here and above?

Reply:
I guess it's fine here?